apps/intellij/src/main/kotlin/dev/nx/console/generate_ui/run_generator/RunGeneratorManager.kt

```diff
@@ -48,28 +54,27 @@ class RunGeneratorManager(val project: Project) {
     }
 
     private fun runGenerator(definition: List<String>) {
-        val pkgManagerExecCommand: List<String> = getPackageManagerExecCommand()
-        val pkgManagerExecutable = pkgManagerExecCommand.get(0)
-        val pkgManagerParameters =
-            pkgManagerExecCommand.let {
-                if (it.size == 1) emptyList() else it.subList(1, it.size)
-            } + definition
-
         try {
             ApplicationManager.getApplication()
                 .invokeLater(
                     {
+                        val binPath =
```

Comment:
I think this should be reworked to also work with [encapsulated nx](https://nx.dev/more-concepts/encapsulated-nx-and-the-wrapper) where there is no `node_modules`

Reply:
yea, might as well support it here now